python/py-pyqt5/Portfile

```diff
@@ -41,6 +44,37 @@ if {${subport} eq "${name}-common"} {
         file copy {*}[glob -directory ${worksrcpath}/sip/ Q*] ${destroot}${prefix}/share/sip/PyQt5
         file copy ${worksrcpath}/sip/Enginio ${destroot}${prefix}/share/sip/PyQt5
     }
+
+} elseif {[string first "webengine" ${subport}] != -1} {
+    PortGroup           qmake5 1.0
+
+    description         PyQt5 Webengine bindings
+    long_description    ${description}
+    master_sites        https://www.riverbankcomputing.com/static/Downloads/PyQtWebEngine/${version}
+    distname            PyQtWebEngine_gpl-${version}
+    checksums           rmd160  5e0c3cd69448b8dee042ba6565c823e266c22a14 \
+                        sha256  860704672ea1b616e1347be1f347bc1c749e64ed378370863fe209e84e9bd473 \
+                        size    42474
+
+    depends_lib-append  port:py${python.version}-pyqt5
+    qt5.depends_component \
+                        qtwebengine
+    use_configure       yes
+    configure.pre_args
+    configure.cmd       "${python.bin} configure.py"
+    qt5.spec_cmd        --spec=
+    configure.args-append \
+                        -q ${qt_qmake_cmd} \
+                        --verbose \
+                        --sip=${prefix}/bin/sip-${python.branch} \
+                        --no-qsci-api \
+                        --no-dist-info
+
+    build.cmd           make
+    build.target        all
+    destroot.cmd        ${build.cmd}
+    destroot.destdir    DESTDIR=${destroot}
```

Comment:
I'm not clear why you need all of these statements. Does the qmake portgroup do anything useful! :)

Reply:
this all comes out of the standard ```py-pyqt5``` port, so your comments do apply to both the ```py-pyqt5``` and ```py-pyqt5-webengine``` ports. To be honest I didn't check if this is all needed,  but if they're the defaults it doesn't hurt either to have them here again... but I would assume there is a reason for it. Perhaps @mamoll can comment on this? Otherwise, in the interest of moving this forward, I'd rather just keep this as is for now and @mamoll can check/change at the next upgrade.

Reply:
Same here.